docs/class-definitions.md

```diff
@@ -423,12 +434,12 @@ Available events:
   space.cursors.subscribe('cursorsUpdate', (cursorUpdate: CursorUpdate) => {});
```

Comment:
Can we please move subscribe to the top of the list after `set` but before the `get..` methods?